package/gluon-status-page/i18n/de.po

```diff
@@ -94,9 +97,15 @@ msgstr "RAM"
 msgid "Received"
 msgstr "Empfangen"
 
+msgid "Role"
+msgstr "Rolle"
+
 msgid "Status"
 msgstr "Status"
 
+msgid "Site"
+msgstr ""
```

Comment:
I'm not sure, but I don't believe this is supposed to be empty string. This would probably override "Site" with nothing, while leaving it out completely would work in our favor. I think you should handle it like "Status" above it.

Reply:
Technically it would work: Empty strings and missing entries are equivalent, meaning "use original string" when used with our usual translate functions, so it would correctly show "Site". We should still keep this consisent with the other entries like "Site", so empty strings are a marker for missing translations.

Please also run msgmerge on fr.po.